src/js/main.js

```diff
@@ -108,8 +115,34 @@ var SupportKit = Marionette.Object.extend({
                     id: res.appUserId
                 });
 
+                var EventCollection = BaseCollection.extend({
+                    url: urljoin('appusers', res.appUserId, 'event'),
+                    model: Event
+                });
+
+                var RuleCollection = Backbone.Collection.extend({
+                    model: Rule
+                });
+
+                // this._events overrides some internals for event bindings in Backbone
+                this._eventCollection = new EventCollection(res.events, {
```

Comment:
In general the either main.js and our controllers need to tolerate all user data (conversation, events, rules) getting swapped out at any time,

or,

every call to init() simply nukes and resets all model+controller state